test/functions/blockchainFunctionsTest.js

```diff
@@ -0,0 +1,95 @@
+require('../helper.js')
+
+const slug = 'santiment'
+const from = new Date(2019, 0, 1, 0, 0, 0)
+const to = new Date(2019, 0, 3, 0, 0, 0)
+const limit = 5
```

Comment:
Also limit is 5, results - 2

Reply:
I've just removed the limit and replaced the limit in the test with 5